Animate gif on profile page

hyenagirl64 opened the pull request:
Second attempt to fix issue #10388

handling it via assigning event handlers in public.js

hyenagirl64 committed:
Quick fix to animate gif avatars on profile pages when hovering

These events should replicate the functionality of gif avatars on other pages at this time.

hyenagirl64 committed:
Quick fix to animate gif avatars on profile pages when hovering

These events should replicate the functionality of gif avatars on other pages at this time.

hyenagirl64 committed:
Quick fix to animate gif avatars on profile pages when hovering

These events should replicate the functionality of gif avatars on other pages at this time.

hyenagirl64 committed:
Merge branch 'animate_gif_profile_2' of https://github.com/hyenagirl64/mastodon into animate_gif_profile_2

hyenagirl64 committed:
Fix so that the gif animation is less likely to cause a conflict by using it's own meta tag

hyenagirl64 committed:
Fix so that the gif animation is less likely to cause a conflict by using it's own meta tag

Gargron committed:
Fix glitched out "not found" message for non-existing profiles in web UI (#10517)

Gargron committed:
Bump version to 2.8.0rc2 (#10501)

Gargron committed:
Fix permission denied bug on approve all/reject all pending accounts (#10519)

dependabot[bot] committed:
Bump bootsnap from 1.4.2 to 1.4.3 (#10523)

Bumps [bootsnap](https://github.com/Shopify/bootsnap) from 1.4.2 to 1.4.3.
- [Release notes](https://github.com/Shopify/bootsnap/releases)
- [Changelog](https://github.com/Shopify/bootsnap/blob/master/CHANGELOG.md)
- [Commits](https://github.com/Shopify/bootsnap/compare/v1.4.2...v1.4.3)

Signed-off-by: dependabot[bot] <support@dependabot.com>

dependabot[bot] committed:
Bump hamlit-rails from 0.2.2 to 0.2.3 (#10522)

Bumps [hamlit-rails](https://github.com/mfung/hamlit-rails) from 0.2.2 to 0.2.3.
- [Release notes](https://github.com/mfung/hamlit-rails/releases)
- [Commits](https://github.com/mfung/hamlit-rails/compare/v0.2.2...v0.2.3)

Signed-off-by: dependabot[bot] <support@dependabot.com>

Gargron committed:
Add "why do you want to join" field to invite requests (#10524)

* Add "why do you want to join" field to invite requests

Fix #10512

* Remove unused translations

* Fix broken registrations when no invite request text is submitted

hinaloe committed:
Allow set the voting period to just 5 minutes (#10525)

* Add spec of PollValidator for #10190

* Raise fraction less than 1 second

* format

* simplify time initialize

yagiz-dev committed:
Fixed terrible grammar and translated the most (#10527)

The translation was written in bad Turkish grammar. I fixed those terrible phrases and translated %93 of Mastodon.

I'll come back and finish the translation.

Gargron committed:
Add preference to disable e-mails about new pending accounts (#10529)

Gargron committed:
Add invite request to pending account notification e-mail (#10528)

Fix sorting of the pending accounts page

Gargron committed:
Change HTML validator to ignore all errors except unmatched tags (#10534)

Gargron committed:
Change the groupings of menu items in settings navigation (#10533)

* Change the groupings of menu items in settings navigation

Fix #10307

* Remove unused translations

Gargron committed:
Bump version to 2.8.0rc3 (#10535)

ThibG commented:
Using a custom opengraph tag to store that value seems weird to me. I think I would have used custom `data-` attributes on the image such as `data-original` and `data-static`. Not too sure about a toggle property but it would probably fit right in there too.

alixrossi committed:
i18n: Update Corsican translations (#10538)

tribela committed:
Translation - Korean (#10540)

koyuawsmbrtn committed:
Updated German translation (#10539)

* Updated German locales

* Added account approval locales (German)

* Updated JSON locales (German)

* Updated German translation

* Fixed grammar error (German)

* New strings and adjustments (German)

marek-lach committed:
Updates the grammar of the SK translation (#10541)

Aditoo17 committed:
I18n: Final Czech translation 🇨🇿 (#10542)

* I18n: Update Czech translation

* I18n: Why do you want to join translation

* I18n: Translating a bunch of more stuff

Aditoo17 committed:
I18n: Add one more string (#10543)

mayaeh committed:
i18n: Update Japanese translations (#10536)

* run i18n-tasks add-missing

* Update Japanese translations.

xgess committed:
compare usernames case-insensitively on new proof creation flow (#10544)

* compare usernames case-insensitively on new proof creation flow

* Fix code style issue

xgess committed:
increase allowable length of remote proof username (#10546)

Gargron committed:
Weblate translations (2019-04-10) (#10545)

* Translated using Weblate (Occitan)

Currently translated at 95.7% (780 of 815 strings)

Translation: Mastodon/Backend
Translate-URL: https://weblate.joinmastodon.org/projects/mastodon/backend/oc/

* Translated using Weblate (Greek)

Currently translated at 100,0% (382 of 382 strings)

Translation: Mastodon/React
Translate-URL: https://weblate.joinmastodon.org/projects/mastodon/frontend/el/

* Translated using Weblate (Greek)

Currently translated at 100,0% (4 of 4 strings)

Translation: Mastodon/Activerecord
Translate-URL: https://weblate.joinmastodon.org/projects/mastodon/activerecord/el/

* Translated using Weblate (Greek)

Currently translated at 100.0% (382 of 382 strings)

Translation: Mastodon/React
Translate-URL: https://weblate.joinmastodon.org/projects/mastodon/frontend/el/

* Translated using Weblate (French)

Currently translated at 100.0% (382 of 382 strings)

Translation: Mastodon/React
Translate-URL: https://weblate.joinmastodon.org/projects/mastodon/frontend/fr/

* Translated using Weblate (French)

Currently translated at 100.0% (4 of 4 strings)

Translation: Mastodon/Activerecord
Translate-URL: https://weblate.joinmastodon.org/projects/mastodon/activerecord/fr/

* Translated using Weblate (French)

Currently translated at 100.0% (66 of 66 strings)

Translation: Mastodon/Devise
Translate-URL: https://weblate.joinmastodon.org/projects/mastodon/devise/fr/

* Translated using Weblate (Dutch)

Currently translated at 100,0% (4 of 4 strings)

Translation: Mastodon/Activerecord
Translate-URL: https://weblate.joinmastodon.org/projects/mastodon/activerecord/nl/

* Translated using Weblate (Galician)

Currently translated at 100,0% (4 of 4 strings)

Translation: Mastodon/Activerecord
Translate-URL: https://weblate.joinmastodon.org/projects/mastodon/activerecord/gl/

* Translated using Weblate (Dutch)

Currently translated at 100,0% (382 of 382 strings)

Translation: Mastodon/React
Translate-URL: https://weblate.joinmastodon.org/projects/mastodon/frontend/nl/

* Translated using Weblate (Galician)

Currently translated at 100,0% (382 of 382 strings)

Translation: Mastodon/React
Translate-URL: https://weblate.joinmastodon.org/projects/mastodon/frontend/gl/

* Translated using Weblate (Slovak)

Currently translated at 100.0% (382 of 382 strings)

Translation: Mastodon/React
Translate-URL: https://weblate.joinmastodon.org/projects/mastodon/frontend/sk/

* Translated using Weblate (Arabic)

Currently translated at 97.4% (372 of 382 strings)

Translation: Mastodon/React
Translate-URL: https://weblate.joinmastodon.org/projects/mastodon/frontend/ar/

* Translated using Weblate (French)

Currently translated at 100.0% (382 of 382 strings)

Translation: Mastodon/React
Translate-URL: https://weblate.joinmastodon.org/projects/mastodon/frontend/fr/

* Translated using Weblate (Thai)

Currently translated at 75.4% (288 of 382 strings)

Translation: Mastodon/React
Translate-URL: https://weblate.joinmastodon.org/projects/mastodon/frontend/th/

* Translated using Weblate (Slovak)

Currently translated at 100.0% (4 of 4 strings)

Translation: Mastodon/Activerecord
Translate-URL: https://weblate.joinmastodon.org/projects/mastodon/activerecord/sk/

* Translated using Weblate (German)

Currently translated at 100,0% (4 of 4 strings)

Translation: Mastodon/Activerecord
Translate-URL: https://weblate.joinmastodon.org/projects/mastodon/activerecord/de/

* Translated using Weblate (Slovak)

Currently translated at 100,0% (4 of 4 strings)

Translation: Mastodon/Activerecord
Translate-URL: https://weblate.joinmastodon.org/projects/mastodon/activerecord/sk/

* Translated using Weblate (Kazakh)

Currently translated at 100,0% (4 of 4 strings)

Translation: Mastodon/Activerecord
Translate-URL: https://weblate.joinmastodon.org/projects/mastodon/activerecord/kk/

* Translated using Weblate (Persian)

Currently translated at 100.0% (382 of 382 strings)

Translation: Mastodon/React
Translate-URL: https://weblate.joinmastodon.org/projects/mastodon/frontend/fa/

* Translated using Weblate (Kazakh)

Currently translated at 99.0% (378 of 382 strings)

Translation: Mastodon/React
Translate-URL: https://weblate.joinmastodon.org/projects/mastodon/frontend/kk/

* i18n-tasks normalize

* yarn manage:translations

* Fix empty string in Occitan

Gargron committed:
Fix Keybase verification using wrong domain for remote accounts (#10547)

MasterGroosha committed:
i18n: Russian localization update (#10515)

* i18n: Updated ru.json for Russian language

* i18n: Updated ru.yml

* Normalize ru.yml

* i18n: Updated simple_form.ru.yml

* i18n: Updated activerecord.ru.yml, devise.ru.yml, normalized strings

* Added new strings to ru.json

* Deleted unused (?) string

* i18n: Added recent strings (Settings refactor)

hyenagirl64 commented:
well, if I'm adding custom attributes, I could store the information as to whether or not to do the hover effect on the image in another attribute, rather than relying on whether or not the toggle is the same as the source.  That would probably be a less bug-prone way to do it as well, since it would make the functionality more explicit.

hyenagirl64 committed:
Less hacky version of this fix

hyenagirl64 committed:
Merge branch 'animate_gif_profile_2' of https://github.com/hyenagirl64/mastodon into animate_gif_profile_2

hyenagirl64 committed:
merge from rebase
